magicblock-accounts-db/src/lib.rs

```diff
@@ -265,8 +265,8 @@ impl AccountsDb {
 
         let used_storage = self.storage.utilized_mmap();
         if let Err(err) = self.snapshot_engine.snapshot(slot, used_storage) {
-            error!(
-                "error taking snapshot at {}, slot {slot}: {err}",
+            warn!(
```

Comment:
Isn't that a serious issue and should be logged on `error`? Or is it expected/ok to happen at times?

Reply:
expected to happen during tests